externs/Cesium.externs.js

```diff
@@ -523,6 +523,13 @@ Cesium.BillboardCollection.prototype.add = function(options) {};
 Cesium.BillboardCollection.prototype.contains = function(opt_bb) {};
 
 
+/**
+ * @param {number} index
+ * @return {Cesium.Billboard} .
+ */
+Cesium.BillboardCollection.prototype.get = function(index) {};
```

Comment:
Why was this needed?

Reply:
It's used by `plugin.cesium.VectorContext` but the old compiler didn't identify that it was missing from externs.